CHANGES.md

```diff
@@ -59,6 +59,10 @@ To be released.
     `IProtocol.CheckReplacementCacheAsync()` to check cached peers
     periodically.  [[#608]]
  -  Marked `Address` and `HashDigest` as readonly.  [[#610]]
+ -  `KademliaProtocol<T>` became to update peer table when receiving messages
```

Comment:
I guess we need to mention it at a higher level.

Reply:
I reworded changes related in this PR, and I'm gonna reword other changes not made by this PR but related to Kademlia protocol in seperated PR soon.